cms/djangoapps/contentstore/views/tests/test_import_export.py

```diff
@@ -697,3 +699,111 @@ def test_content_library_export_import(self):
+    """
+    def setUp(self):
+        super(TestCourseExportImport, self).setUp()
+        self.export_dir = tempfile.mkdtemp()
+
+        # Create a problem in library
+        ItemFactory.create(
+            category="problem",
+            parent_location=self.library.location,
+            user_id=self.user.id,
+            publish_item=False,
+            display_name='Test Problem',
+            data="<problem><multiplechoiceresponse></multiplechoiceresponse></problem>",
+        )
+
+        # Create a source course.
+        self.source_course = CourseFactory.create(default_store=ModuleStoreEnum.Type.split)
+        self.addCleanup(shutil.rmtree, self.export_dir, ignore_errors=True)
+
+    def _setup_source_course_with_library_content(self, publish=False):
+        """
+        Sets up course with library content.
+        """
+        chapter = ItemFactory.create(
+            parent_location=self.source_course.location,
+            category='chapter',
+            display_name='Test Section'
+        )
+        sequential = ItemFactory.create(
+            parent_location=chapter.location,
+            category='sequential',
+            display_name='Test Sequential'
+        )
+        vertical = ItemFactory.create(
+            category='vertical',
+            parent_location=sequential.location,
+            display_name='Test Unit'
+        )
+        lc_block = self._add_library_content_block(vertical, self.lib_key, publish_item=publish)
+        self._refresh_children(lc_block)
+
+    def get_lib_content_block_children(self, block_location):
+        """
+        Search for library content block to return its immediate children
+        """
+        if block_location.block_type == 'library_content':
+            return self.store.get_item(block_location).children
+        else:
+            return self.get_lib_content_block_children(self.store.get_item(block_location).children[0])
+
+    def assert_problem_display_names(self, source_course_location, dest_course_location):
+        """
+        Asserts that problems' display names in both source and destination courses are same.
+        """
+        source_course_lib_children = self.get_lib_content_block_children(source_course_location)
+        dest_course_lib_children = self.get_lib_content_block_children(dest_course_location)
+
+        self.assertEquals(len(source_course_lib_children), len(dest_course_lib_children))
+
+        for source_child_location, dest_child_location in zip(source_course_lib_children, dest_course_lib_children):
+            source_child = self.store.get_item(source_child_location)
+            dest_child = self.store.get_item(dest_child_location)
+            self.assertEquals(source_child.display_name, dest_child.display_name)
+
+    @ddt.data(True, False)
+    def test_library_content_on_course_export_import(self, publish_item):
+        """
+        Asserts library content in destination and source courses after importing
```

Comment:
Should start with the `Test that ... ` or `Check that ... `. This looks ambiguous with `assert_problem_display_names` method which also asserts but is not a test.